<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEventWidget</class>
 <widget class="QWidget" name="QEventWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Precondition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkSwitchA">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineSwitchA">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="toolSwitchA">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkSwitchB">
          <property name="text">
           <string>Switch</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineSwitchB">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="toolSwitchB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkVar">
          <property name="text">
           <string>Variable</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineVar">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="toolVar">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="comboVarOperation">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Equal to                     (==)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Greather or equal to (&gt;=)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lesser or equal to     (&lt;=)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Greather than             (&gt;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lesser than                 (&lt;)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Not equal to              (!=)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinVarValue">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>-999999</number>
          </property>
          <property name="maximum">
           <number>999999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="checkItem">
          <property name="text">
           <string>Item</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboItem">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkHero">
          <property name="text">
           <string>Hero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboHero">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkTimerA">
          <property name="text">
           <string>Timer 1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinTimerAMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> Minutes</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinTimerASec">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string>  seconds or less</string>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkTimerB">
          <property name="text">
           <string>Timer 2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinTimerBMin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> Minutes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinTimerBSec">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="suffix">
           <string> seconds or less</string>
          </property>
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" rowspan="5">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>3</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Event Commands</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QTreeWidget" name="treeCommands">
        <column>
         <property name="text">
          <string>Command</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Id</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Event Graphic</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsSprite">
        <property name="maximumSize">
         <size>
          <width>56</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QCheckBox" name="checkTransparent">
          <property name="text">
           <string>Trans</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushSetSprite">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Trigger Condition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboCondition">
        <item>
         <property name="text">
          <string>Action Key</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Touched by Hero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Collision with Hero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AutoStart</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Parallel Process</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Event Layer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboLayer">
        <item>
         <property name="text">
          <string>Bellow Hero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Same as Hero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Above Hero</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkOverlap">
        <property name="text">
         <string>Forbid overlap</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Movement Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboMoveType">
        <item>
         <property name="text">
          <string>Stationary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Random</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vertical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Toward Hero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Away from Hero</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom Pattern</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboMoveSpeed">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushMovePattern">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Define Pattern</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>Animation type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboAnimationType">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Non-Continuos</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Continuous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fixed Dir/Non-Countinuous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fixed Dir/Countinuous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fixed Graphic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spin Around</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_10">
     <property name="title">
      <string>Move Frequency</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboMoveFrequency">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <item>
         <property name="text">
          <string>1: (1/8) Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2: (1/4) Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3: (1/2) Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4: Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5: Normal x 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6: Normal x 4</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkSwitchA</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineSwitchA</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSwitchB</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineSwitchB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkVar</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineVar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkVar</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolVar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSwitchB</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolSwitchB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSwitchA</sender>
   <signal>toggled(bool)</signal>
   <receiver>toolSwitchA</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkVar</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinVarValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkVar</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboVarOperation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkItem</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboItem</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkHero</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboHero</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerA</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerAMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerA</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerASec</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerB</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerBMin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkTimerB</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTimerBSec</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
